speedy-recovery-client/src/components/secured/home/HomePageFactory.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import HomePageForPatient from "../home/HomePageForPatient";
+import HomePageForParent from "../home/HomePageForParent";
+import HomePageForPractitioner from "../home/HomePageForPractitioner";
+
+class HomePageFactory extends Component {
+  render() {
+    const { user, patient, events } = this.props;
+
+    switch (user.role) {
+      case "Patient":
+        return (
+          <HomePageForPatient
```

Comment:
Can you please rename them into `PatientHomePage` etc for all roles? Remember too also change the Factory.test.js